### PR TITLE
Add WildFly 29 documentation

### PR DESCRIPTION
**Pull request by bstansberry**

### CODE REVIEW

File: sitemap.xml [hunk 1]
```diff
@@ -1,11 +1,8 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
+29<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
```
Comment (fjuma) [inline sitemap.xml:1]: Should the 29 here be removed?

Comment (bstansberry) [reply sitemap.xml:1]: Yes; thanks!

Comment (bstansberry) [reply sitemap.xml]: @fjuma Fixed.

**Review by fjuma — APPROVED** [review]
Thanks @bstansberry!

**Pull request merged**